crd-generator/src/main/java/io/fabric8/crd/generator/apt/CustomResourceAnnotationProcessor.java

```diff
@@ -115,18 +115,15 @@ private CustomResourceInfo add(TypeElement element) {
         System.out.println("Ignoring " + crClassName + " because it isn't parameterized");
         return null;
       }
-      final TypeElement spec = ((TypeElement) ((DeclaredType) typeArguments.get(0)).asElement());
       final TypeElement status = ((TypeElement) ((DeclaredType) typeArguments.get(1)).asElement());
-      final CustomResourceInfo info = new CustomResourceInfo(element, spec, status, processingEnv);
+      final CustomResourceInfo info = new CustomResourceInfo(element, status, processingEnv);
       System.out.println(
         "Generating '"
           + info.crdName()
           + "' version '"
           + info.version()
           + "' with "
           + crClassName
-          + " (spec: "
```

Comment:
The field is indeed not used but might be at some point to inform the CRD generation. However, even if the field is not used, the logging that informs the users about which CRD is being generated from which CR shouldn't be removed. So, at the very least, this needs to stay.

Reply:
@metacosm : Sorry for the late reply
I added the logging field. Please take a look. Thanks